Dockerfile

```diff
@@ -3,28 +3,37 @@
 FROM debian:12.2-slim as builder
 
 # Install needed packages
-RUN apt update -y && \
-    apt install build-essential git clang curl libssl-dev llvm libudev-dev make cmake protobuf-compiler -y
+RUN apt-get update -y && \
+    apt-get install build-essential \
+		git \
+		clang \
+		curl \
+		libssl-dev \
+		llvm \
+		libudev-dev \
+		make \
+		cmake \
+		protobuf-compiler -y --no-install-recommends
 
 WORKDIR "/da/src"
 
 # Clone repo
-ARG AVAIL_TAG=v1.8.0.3
+ARG AVAIL_TAG=v1.8.0.4
```

Comment:
We don't need to update the TAG version here. This is injected during build time.